utils/Database.py

```diff
@@ -0,0 +1,165 @@
+import configparser
+import logging
+from pathlib import Path
+from typing import Callable, Concatenate, ParamSpec, Sequence, TypeVar
+
+from sqlalchemy import Column, Integer, String, create_engine, select
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+logger = logging.getLogger(__name__)
+
+
+Base = declarative_base()
+
+
+class EmailClaimedError(Exception):
+    pass
+
+
+class User(Base):
+    __tablename__ = "users"
+    email = Column(String, primary_key=True)
+    discord_id = Column(Integer, nullable=True)
+    todoist_id = Column(String)
+    todoist_token = Column(String)
+
+    def __repr__(self) -> str:
+        return f"<User(email={self.email}, discord_id={self.discord_id}, todoist_id={self.todoist_id})>"
+
+
+def migrate_from_cfg(config: configparser.ConfigParser) -> list[User]:
```

Comment:
thoughts on moving the config -> database migration code into a seperate script? 

is the intent to remove this logic after we run this in the server?

Reply:
yeah I suppose this should live in a "migrations" script with tracked versions. this was mainly a concept test